pkg/fleet/installer/setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+# This script is used to install the Datadog.
+# Note to developer: This script is only responsible of starting the actual installer binary.
+
+if [ "$(uname -s)" != "Linux" ] || { [ "$(uname -m)" != "x86_64" ] && [ "$(uname -m)" != "aarch64" ]; }; then
```

Comment:
there's at least one different value of uname -m supported `amd64`
I believe arm64 is also a possible output but couldn't find in telemetry

Reply:
checked on slack, amd64 does not actually appear in telem